brd-ios/breadwallet/src/Helpers/UITextField+Extensions.swift

```diff
@@ -0,0 +1,13 @@
+//
+// Created by Equaleyes Solutions Ltd
+//
+
+import UIKit
+
+extension UITextField {
+    func setPasswordToggleImage(_ button: UIButton) {
```

Comment:
this does not need to be an extension. you can have this in the text field class.